tests/certification/pubsub/azure/servicebus/components/authentication/service_bus.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: messagebus
+  namespace: default
+spec:
+  type: pubsub.azure.servicebus
+  version: v1
+  metadata:
+    - name: consumerID
+      value: optionscertification
+    - name: namespaceName
+      secretKeyRef:
+        name: AzureServiceBusNamespace
```

Comment:
Good, but where is this coming from if you didn't actually add it in the workflow :)

Reply:
Got it!!!